src/components/bookReact.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+
+const dummyBooks = [
+  {
+    id: 123,
+    title: 'This is a book title',
+    auhor: 'Alex',
+  },
+];
+
+const DisplayBook = () => {
+  const [book, setBook] = useState(dummyBooks);
+  return (
+    <div>
+      <section>
+        <p>{book[0].title}</p>
+        <ul>
+          <li>Comments</li>
+          <li>Remove</li>
+          <li>Edit</li>
+        </ul>
+      </section>
+      <section>
+        <form>
+          <h2>Add a new book</h2>
+          <input type="text" placeholder="Book Title" />
+          <select name="Categories">
+            <option>Aciton</option>
+            <option>Races</option>
+          </select>
+          <input type="submit" value="Add Book" onClick={() => setBook((book) => [...book])} />
+        </form>
+      </section>
```

Comment:
- [ ] Kindly extract this form for adding a new book into a separate component.

Reply:
Yes, thank you for your comments, I'll refactor both points 😃  🧑‍💻